docs/NativeModulesIOS.md

```diff
@@ -180,6 +180,8 @@ CalendarManager.findEvents((error, events) => {
 
 A native module is supposed to invoke its callback only once. It can, however, store the callback and invoke it later. This pattern is often used to wrap iOS APIs that require delegates. See [`RCTAlertManager`](https://github.com/facebook/react-native/blob/master/React/Modules/RCTAlertManager.m) for an example.
 
+For an alternative mechanism for sending data to JavaScript, see [Sending Events to JavaScript](#sending-events-to-javascript). You can use the mechanism to send data twice or more, except for some events should be fired once, for example, `onReady`.
```

Comment:
`#sending-events-to-javascript` seems to be invalid? I can't find it in the project.